tests/NuGet.Services.AzureSearch.Tests/Auxiliary2AzureSearch/UpdateDownloadsCommandFacts.cs

```diff
@@ -522,8 +522,12 @@ public Facts(ITestOutputHelper output)
                         PushedIds.Add(id);
                     });
                 BatchPusher
-                    .Setup(x => x.FinishAsync())
-                    .Returns(Task.Delay(TimeSpan.FromMilliseconds(1)))
+                    .Setup(x => x.TryFinishAsync())
+                    .Returns(async () =>
+                    {
+                        await Task.Delay(TimeSpan.FromMilliseconds(1));
```

Comment:
What's the sleep for? Could you instead return a `Task.FromResult(...)`?

Reply:
Pre-existing. Intention is to have a bit of time to assert against in a duration metric.

Reply:
Oh there's a comment on line 513 explaining this. My bad :)